bakery/src/cli/execute.js

```diff
@@ -499,8 +495,8 @@ const yargs = require('yargs')
         handler(argv).catch((err) => { console.error(err) })
       }
     }
-  }).call())
-  .command((() => {
+  },
+  checksum: (parentCommand) => {
```

Comment:
Minor (and not directly related to this PR): Is the comment below on line 500 intended to be there, or can it be cleaned up?